app-web/carbons/autobuild/build

```diff
@@ -0,0 +1,5 @@
+abinfo "Building carbons ..."
+make FLAGS="$CFLAGS"
+
+abinfo "Installing carbons ..."
+make install DESTDIR="$PKGDIR"
```

Comment:
Is `PURPLE_PLUGIN_DIR` specified?

Reply:
The path is no longer required to pass. The default value is good.

Reply:
Actually, the previous path was wrong after upstream added $(DESTDIR):

```
carbons upgraded from 0.2.3 to 0.2.3-1 on arm64:
<details>
<summary>3 added, 11 removed</summary>

```diff
--- a
+++ b
@@ -1,17 +1,9 @@
 drwxr-xr-x     ./
 drwxr-xr-x     ./usr/
+drwxr-xr-x     ./usr/lib/
+drwxr-xr-x     ./usr/lib/purple-2/
+-rw-r--r--     ./usr/lib/purple-2/carbons.so
 drwxr-xr-x     ./usr/share/
 drwxr-xr-x     ./usr/share/doc/
 drwxr-xr-x     ./usr/share/doc/carbons/
 -rw-r--r--     ./usr/share/doc/carbons/LICENSE
-drwxr-xr-x     ./var/
-drwxr-xr-x     ./var/cache/
-drwxr-xr-x     ./var/cache/acbs/
-drwxr-xr-x     ./var/cache/acbs/build/
-drwxr-xr-x     ./var/cache/acbs/build/acbs.qfhlh6s3/
-drwxr-xr-x     ./var/cache/acbs/build/acbs.qfhlh6s3/carbons-0.2.3/
-drwxr-xr-x     ./var/cache/acbs/build/acbs.qfhlh6s3/carbons-0.2.3/abdist/
-drwxr-xr-x     ./var/cache/acbs/build/acbs.qfhlh6s3/carbons-0.2.3/abdist/usr/
-drwxr-xr-x     ./var/cache/acbs/build/acbs.qfhlh6s3/carbons-0.2.3/abdist/usr/lib/
-drwxr-xr-x     ./var/cache/acbs/build/acbs.qfhlh6s3/carbons-0.2.3/abdist/usr/lib/purple-2/
--rw-r--r--     ./var/cache/acbs/build/acbs.qfhlh6s3/carbons-0.2.3/abdist/usr/lib/purple-2/carbons.so

```